docs/DEVELOPMENT.md

```diff
@@ -39,12 +45,33 @@ make install && make deploy
 kubectl create -k config/samples/ccruntime/default
 ```
 
-## Uninstalling Operator
+If you have a custom payload image stored in your local registry (for example, `localhost:5000/build-kata-deploy`), the following set of commands is applicable:
 
-Ensure KUBECONFIG points to target Kubernetes cluster
 ```
-make uninstall && make undeploy
+PAYLOAD_IMG=localhost:5000/build-kata-deploy
+pushd config/samples/ccruntime/default
+kustomize edit set image quay.io/kata-containers/kata-deploy=${PAYLOAD_IMG}
+kubectl create -k .
+popd
+```
+
+Additionally, there are alternative patch layers such as `peer-pods` or `s390x` in addition to the `default` layer. You may switch the directory accordingly based on your requirements.
```

Comment:
hi @BbolroC !

Not sure if you really meant `patch layers` or `path layers`. In any case, kustomize uses the jargon "overlay" to mean those directories, i.e., those are "kustomize overlay directories".